internal/subshell/sscommon/rcfile.go

```diff
@@ -392,8 +412,6 @@ func SetupProjectRcFile(prj *project.Project, templateName, ext string, env map[
 		return nil, errs.Wrap(err, "Failed to write to output buffer.")
 	}
 
-	logging.Debug("Using project RC: (%s) %s", tmpFile.Name(), o.String())
-
```

Comment:
Did you mean to leave this deleted?

Reply:
Yes, this shouldn't have been committed way back when, I'm trying to over time reduce the log file down to the helpful essentials.